docs/faq.rst

```diff
@@ -120,6 +122,70 @@ decision about whether you want to use compression, which algorithm
 and which level you want to use. This is why compression defaults to
 none.
 
+If a backup stops mid-way, does the already-backed-up data stay there?
+----------------------------------------------------------------------
+
+Yes, |project_name| supports resuming backups.
```

Comment:
Reminds me I need to rephrase that. In fact, we never resume a backup, we always do full backups from the beginning.
It just feels like resuming...